docs/dev-notes/2026-04-15/relative_evalution_voting/plan.md

```
@@ -0,0 +1,30 @@
+# 概要
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)